fair_mango/metrics/base.py

```diff
@@ -10,17 +10,17 @@
 
 
 def is_binary(y: pd.Series | pd.DataFrame) -> bool:
-    """Check if a data contains two unique values
+    """Check if a data contains two unique values.
 
     Parameters
     ----------
     y : pd.Series | pd.DataFrame
-        data
+        Input data.
 
     Returns
     -------
     bool
-        true if data contains two unique values else false
+        True if data contains two unique values else False.
     """
     try:
```

Comment:
nope nope nope. Don't use try: except: instead of if statements because you are lazy.

Reply:
This is already resolved in a separate branch because the issue of out of scope for this PR.